pandas/tests/io/formats/test_to_string.py

```diff
@@ -107,37 +107,51 @@ def test_format_remove_leading_space_dataframe(input_array, expected):
 
 
 @pytest.mark.parametrize(
-    "max_cols, expected",
+    "max_cols, max_rows, expected",
     [
         (
             10,
-            [
-                " 0   1   2   3   4   ...  6   7   8   9   10",
-                "  0   0   0   0   0  ...   0   0   0   0   0",
-                "  0   0   0   0   0  ...   0   0   0   0   0",
-            ],
+            None,
+            " 0   1   2   3   4   ...  6   7   8   9   10\n"
```

Comment:
is there really a shift on a row with bigger column numbers?

Reply:
Sorry, I'm not sure I understand the question.

Reply:
meaning the first column you are showing (e.g. the 10 at the end) is shifted 1 space to the left (at least it appears that way)

Reply:
If I'm following you correctly, I think it just looks that way because I'm no longer storing the expected output in a list, but instead storing it in a single string across multiple lines.

Reply:
If you're talking about the column headers where the column name is a single character (i.e. 0-9 in this scenario), and the column name is left-justified which leaves a space directly above each 0 in the first row of the data: I think looks a little funky, but this is the standard behavior even with `index=True`.

```python
pd.DataFrame([[0]*11]*2)
```

```
   0   1   2   3   4   5   6   7   8   9   10
0   0   0   0   0   0   0   0   0   0   0   0
1   0   0   0   0   0   0   0   0   0   0   0
```

Reply:
ahh because its left justified, got it